---
layout: single
title: Omics Data Analysis
author_profile: true
---


Supervised multiblock analysis (https://khliland.github.io/multiblock/articles/vignette_E_supervised.html) - https://www.jstatsoft.org/article/view/v086i01


# Benchmark

Ferrarini A, Righetti L, Martínez MP, Fernández-López M, Mastrangelo A, Horcajada JP, Betbesé A, Esteban A, Ordóñez J, Gea J, Cabello JR, Pellati F, Lorente JA, Nin N, Rupérez FJ. Discriminant biomarkers of acute respiratory distress syndrome associated to H1N1 influenza identified by metabolomics HPLC-QTOF-MS/MS platform. Electrophoresis. 2017 Sep;38(18):2341-2348. doi: 10.1002/elps.201700112. Epub 2017 Aug 25. PMID: 28714069.

Annual Review of Statistics and Its Application Volume 2, 2015, Review Article, Microbiome, Metagenomics, and High-Dimensional Compositional Data Analysis, Hongzhe Li1, Vol. 2:73-94 (Volume publication date May 2015) https://doi.org/10.1146/annurev-statistics-010814-020351

Kelly RS, McGeachie MJ, Lee-Sarwar KA, Kachroo P, Chu SH, Virkud YV, Huang M, Litonjua AA, Weiss ST, Lasky-Su J. Partial Least Squares Discriminant Analysis and Bayesian Networks for Metabolomic Prediction of Childhood Asthma. Metabolites. 2018 Oct 23;8(4):68. doi: 10.3390/metabo8040068. PMID: 30360514; PMCID: PMC6316795. >>> A+ good study

Fan S, Shahid M, Jin P, Asher A, Kim J. Identification of Metabolic Alterations in Breast Cancer Using Mass Spectrometry-Based Metabolomic Analysis. Metabolites. 2020 Apr 24;10(4):170. doi: 10.3390/metabo10040170. PMID: 32344578; PMCID: PMC7241246. >>> good


# Feature selection 

Wieder, C., Lai, R.P.J. & Ebbels, T.M.D. Single sample pathway analysis in metabolomics: performance evaluation and application. BMC Bioinformatics 23, 481 (2022). https://doi.org/10.1186/s12859-022-05005-1


Mahajan, P., Fiehn, O. & Barupal, D. IDSL.GOA: gene ontology analysis for interpreting metabolomic datasets. Sci Rep 14, 1299 (2024). https://doi.org/10.1038/s41598-024-51992-x

Processing and Visualization of Metabolomics Data Using R, Written By Stephen C. Grace and Dane A. Hudson, Submitted: 29 March 2016 Reviewed: 29 August 2016 Published: 28 December 2016, DOI: 10.5772/65405


https://math.arizona.edu/~hzhang/waeso/vsTutorial.pdf

Sanchez-Pinto LN, Venable LR, Fahrenbach J, Churpek MM. Comparison of variable selection methods for clinical predictive modeling. Int J Med Inform. 2018 Aug;116:10-17. doi: 10.1016/j.ijmedinf.2018.05.006. Epub 2018 May 21. PMID: 29887230; PMCID: PMC6003624.

Ghazaleh Aliakbarzadeh, Hadi Parastar, Hassan Sereshti, Classification of gas chromatographic fingerprints of saffron using partial least squares discriminant analysis together with different variable selection methods, Chemometrics and Intelligent Laboratory Systems, Volume 158, 2016, Pages 165-173, ISSN 0169-7439, https://doi.org/10.1016/j.chemolab.2016.09.002. (https://www.sciencedirect.com/science/article/pii/S0169743916302969)


https://topepo.github.io/caret/variable-importance.html
https://www.r-bloggers.com/2013/08/variable-importance-in-neural-networks/


Kirpich A, Ainsworth EA, Wedow JM, Newman JRB, Michailidis G, McIntyre LM. Variable selection in omics data: A practical evaluation of small sample sizes. PLoS One. 2018 Jun 21;13(6):e0197910. doi: 10.1371/journal.pone.0197910. PMID: 29927942; PMCID: PMC6013185.

https://www.cs.man.ac.uk/~nogueirs/files/ecml2016.pdf

Erban A, Fehrle I, Martinez-Seidel F, Brigante F, Más AL, Baroni V, Wunderlin D, Kopka J. Discovery of food identity markers by metabolomics and machine learning technology. Sci Rep. 2019 Jul 4;9(1):9697. doi: 10.1038/s41598-019-46113-y. PMID: 31273246; PMCID: PMC6609671.




# pathways 

https://github.com/XikunHan/metabolomicsR

Clauset, A., Moore, C. & Newman, M. Hierarchical structure and the prediction of missing links in networks. Nature 453, 98–101 (2008). https://doi.org/10.1038/nature06830

https://metpath.tidymass.org/

Shen, X., Yan, H., Wang, C. et al. TidyMass an object-oriented reproducible analysis framework for LC–MS data. Nat Commun 13, 4365 (2022). https://doi.org/10.1038/s41467-022-32155-w

https://www.metabolomicsworkbench.org/tools/externaltools.php


Kamburov A, Cavill R, Ebbels TM, Herwig R, Keun HC. Integrated pathway-level analysis of transcriptomics and metabolomics data with IMPaLA. Bioinformatics. 2011 Oct 15;27(20):2917-8. doi: 10.1093/bioinformatics/btr499. Epub 2011 Sep 4. PMID: 21893519.

Chen, Y., Li, E. M., & Xu, L. Y. (2022). Guide to Metabolomics Analysis: A Bioinformatics Workflow. Metabolites, 12(4), 357. https://doi.org/10.3390/metabo12040357

https://encyclopedia.pub/entry/23325

MetENP/MetENPWeb: An R package and web application for metabolomics enrichment and pathway analysis in Metabolomics Workbench
Kumari Sonal Choudhary, Eoin Fahy, Kevin Coakley, Manish Sud, Mano R Maurya, Shankar Subramaniam
bioRxiv 2020.11.20.391912; doi: https://doi.org/10.1101/2020.11.20.391912


Uppal K, Soltow QA, Promislow DE, Wachtman LM, Quyyumi AA, Jones DP. MetabNet: An R Package for Metabolic Association Analysis of High-Resolution Metabolomics Data. Front Bioeng Biotechnol. 2015 Jun 11;3:87. doi: 10.3389/fbioe.2015.00087. PMID: 26125020; PMCID: PMC4464066.

Uppal K, Soltow QA, Promislow DE, Wachtman LM, Quyyumi AA, Jones DP. MetabNet: An R Package for Metabolic Association Analysis of High-Resolution Metabolomics Data. Front Bioeng Biotechnol. 2015 Jun 11;3:87. doi: 10.3389/fbioe.2015.00087. PMID: 26125020; PMCID: PMC4464066.

Tsouka, S., & Masoodi, M. (2023). Metabolic Pathway Analysis: Advantages and Pitfalls for the Functional Interpretation of Metabolomics and Lipidomics Data. Biomolecules, 13(2), 244. https://doi.org/10.3390/biom13020244